com.microsoft.java.debug.core/src/main/java/com/microsoft/java/debug/core/Breakpoint.java

```diff
@@ -149,6 +173,7 @@ public void setHitCount(int hitCount) {
             .filter(request -> request instanceof BreakpointRequest)
             .subscribe(request -> {
                 request.addCountFilter(hitCount);
+                request.disable();
```

Comment:
why we need to call `disable` first here?

Reply:
![image](https://user-images.githubusercontent.com/14052197/191459813-2765c921-72cc-4bbf-8752-563167b0fd8f.png)

Calling request.enable() will do nothing if the request has been enabled before. We need to invalidate the old request, and re-enable it to send the new JDWP request to debuggee.

https://github.com/eclipse-jdt/eclipse.jdt.debug/blob/96cda748fde8d874fff6e3fad1f3f775377eac93/org.eclipse.jdt.debug/jdi/org/eclipse/jdi/internal/request/EventRequestImpl.java#L245-L248